src/Mod/Part/Gui/SectionCutting.cpp

```diff
@@ -110,9 +110,9 @@ SectionCut::SectionCut(QWidget* parent)
         return;

```

Comment:
The "ui" member is a raw pointer and not deleted. It's best to convert it into a std::unique_ptr.

Reply:
Thanks, I fixed this now.